policyengine_us/parameters/gov/states/ny/tax/income/supplemental/incremental_benefit/widow.yaml

```diff
@@ -5,33 +5,39 @@ metadata:
   amount_unit: currency-USD
   label: New York supplemental tax incremental benefit widow amount
   reference:
-    - title: Section 601 Imposition of tax, (d-2) for 2021 and (d-3) for 2022
+    - title: Section 601 Imposition of tax, (d-2) for 2021, (d-3) for 2022, and (d-4) for 2023
       href: https://www.nysenate.gov/legislation/laws/TAX/601
     - title: 2021 NY Form IT-201 Instructions - Married filing jointly and qualifying widow(er)
       href: https://www.tax.ny.gov/pdf/2021/inc/it201i_2021.pdf#page=52
     - title: 2022 NY Form IT-201 Instructions - Married filing jointly and qualifying surviving spouse
       href: https://www.tax.ny.gov/pdf/2022/printable-pdfs/inc/it201i-2022.pdf#page=45
+    - title: 2023 NY Form IT-201 Instructions - Married filing jointly and qualifying surviving spouse
+      href: https://www.tax.ny.gov/pdf/2023/printable-pdfs/inc/it201i-2023.pdf#page=50
```

Comment:
not sure if this is the right page for these parameters -> please double check all yaml